1-Draft/RFCXXXX-Policy.md

```diff
@@ -0,0 +1,261 @@
+---
+RFC:  RFCnnnn
+Author:  travisez13
+Status:  Draft
+SupercededBy:  N/A
+Version: 0.1
+Area: Engine
+Comments Due: 6/30/2019
+---
+
+# `PowerShell Core` Policy
+
+## Motivation
+
+Consumers, developers, and enterprise system administrators should be able to flexibly and reliably configure PowerShell 7.
+
+## Acknowledgement
+
+I based this off of @iSazonov 's RFC, for just a slightly different purpose.
+[PR #111](https://github.com/PowerShell/PowerShell-RFC/pull/111)
+
+## Goals
+
+1. Specify how PowerShell 7 will deal with having both Windows PowerShell and PowerShell Core Group Policy.
+    - This is covered in [Policy settings Setting Fall-Back](#policy-settings-setting-fall-back).
+1. Define how the `pwsh -settingsfile` switch should behave.
+    - This is covered in  [Parameter `-settingsfile`](#Parameter--settingsfile)
+
+## Definitions
+
+- **Computer-Wide settings/policy**  - setting or policy applied to an operating system environment (OSE), affecting all users of the OSE.
+- **User settings/policy** - setting or policy applied to a specific user of an OSE, and not applied to the OSE as a whole.
+
+## Specification
+
+`PowerShell 7` should be configured using the following schemes:
+
+- On Windows - Group Policy Objects (GPO), and settings files.
+- On Unix - settings files.
+
+The settings files have `JSON` format.
+
+**Warning** The settings files differ from `PowerShell 7` _profile_ files, which are PowerShell scripts run at startup.
+
+Configuration schemes allow to customize `PowerShell 7` in the most flexible way:
+
+- Enterprise system administrators can use GPO,
+    GPP and computer-wide settings files to apply approved configuration settings and mandatory security settings in a centralized manner.
+    Most settings can be applied either to the user or computer-wide.
+    We will cover the precedence of these in [Registry keys and settings](#registry-keys-and-settings).
+- Developers and consumers can use user, or computer-wide level setting files.
+
+### Configuration defaults
+
+PowerShell 7 has hard-coded defaults for all configuration policies and settings.
+
+The default values must be `secure-by-default`.
+
+For release versions hard-coded defaults must be the same as ones in pre-installed configuration files. For preview versions they may vary (ex., enable experimental features and so on).
+
+Computer-wide configuration includes security sensitive setting,
+and failing to read those setting, for example if the file is locked or corrupted, could result in an insecure system.
+So, if during startup, PowerShell 7 cannot read settings files from the Computer-Wide scope,
+it fails to startup.
```

Comment:
We cannot have secure by default when encryption is involved.   We cannot securely encrypt and have a way for people to decrypt.

Reply:
> it fails to startup

I agree with the behavior in lock-down systems but not by default. 
As I said above, this can cause unpredictable behavior - users will be at a loss as to why their application is not launched at random.